### PR TITLE
PEPPER-460 os2 somatic ped v3

### PR DESCRIPTION
**Pull request by ssettipalli**
## Context
Osteo somatic consent addendum pediatric (consent portion only) pe-cgs updates

## FUD Score

_Overall, how are you feeling about these changes?_

- [x ] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ x] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [x ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/osteo/patches/somatic-consent-addendum-ped-v3.conf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                "name": "osteo_consent_somatic_contact_p_pediatric",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": """If you have questions about the study, please contact the research doctor or study staff listed below by emailing  <a href="mailto:info@osproject.org" class="Link">info@osproject.org</a> or calling <a href="tel:651-602-2020" class="Link">651-602-2020</a>:"""
+                  }
+                ]
+              },
+              {
+                "name": "osteo_consent_somatic_contact_li2_pediatric",
```
Comment (otchet-broad) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/somatic-consent-addendum-ped-v3.conf:46]: Is li2 the right variable name?  If I understand correctly, there's only one entry in the list now.

**Review by otchet-broad — APPROVED** [review]
I think there's one small change I would make (variable name), but overall LGTM!

File: pepper-apis/studybuilder-cli/studies/osteo/patches/somatic-consent-addendum-ped-v3.conf [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                ]
+              }
+            ]
+          },
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """
+                        <p>$osteo_consent_somatic_contact_p_pediatric</p>
+                        <ul>
+                           <li>$osteo_consent_somatic_contact_li2_pediatric</li>
```
Comment (otchet-broad) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/somatic-consent-addendum-ped-v3.conf:31]: See note below about variable name.

File: pepper-apis/studybuilder-cli/studies/osteo/patches/somatic-consent-addendum-ped-v3.conf (hunk 1 above)
Comment (ssettipalli) [reply pepper-apis/studybuilder-cli/studies/osteo/patches/somatic-consent-addendum-ped-v3.conf:46]: Thats the actual variable name for needed text and I dont want to change the name and complicate it.

**Pull request merged**